### DISCUSSION TITLE
PackageNotFoundError: TTS when using Windows Subsystem for Linux

### DISCUSSION THREAD

**Discussion by John2Kelland** [coqui-ai/TTS#1259] [General Q&A]
Hello,

I am new to utilizing the coqui-ai TTS solution, but I have been successfully generating audio on a Linux VM.

When I install coqui-ai using Windows Subsystem for Linux, however, I receive the following error:
![image](https://user-images.githubusercontent.com/87883949/154537407-d37419a3-db11-4e22-9312-27957e8e18a6.png)

I noticed that /usr/bin/tts does not even exist on my Linux VM, so I'm not sure why TTS is failing in WSL.

The Linux VM and my instance of WSL are both using Python 3.8.10 and Ubuntu 20.04.

Any ideas are appreciated, and thank you in advance!

**Comment by fijipants** [comment]
Have you tried with a Conda environment?

**Reply by John2Kelland** [reply]
What would the approach for a resolution be with a Conda environment?
